ops-bedrock/docker-compose.yml

```diff
@@ -80,7 +80,7 @@ services:
       --p2p.listen.ip=0.0.0.0
       --p2p.listen.tcp=9003
       --p2p.listen.udp=9003
-      --p2p.scoring.peers=light
+      --p2p.scoring=light
```

Comment:
Is this a correct change?

Reply:
no it appears wrong to me: https://github.com/Layr-Labs/optimism/blob/55a63526357cf8c672e996896baf66328fcb4563/op-node/flags/p2p_flags.go#L20